.github/workflows/e2e.yaml

```diff
@@ -93,35 +93,12 @@ jobs:
         # output_dir
         mkdir -p _artifacts
         # create cluster
-        cat <<EOF | /usr/local/bin/kind create cluster \
+        sed -i  's/IP_FAMILY/'${IP_FAMILY}'/g' examples/kind/kind.config
+        sed -i  's#EGREE_SELECTOR_CONFIGURATION_PATH#./examples/kind/egress_selector_configuration.yaml#g' examples/kind/kind.config
+        /usr/local/bin/kind create cluster \
           --name ${{ env.KIND_CLUSTER_NAME}}           \
           --image kindest/node:${{ env.K8S_VERSION }}  \
-          -v7 --wait 1m --retain --config=-
-        kind: Cluster
-        apiVersion: kind.x-k8s.io/v1alpha4
-        networking:
-          ipFamily: ${IP_FAMILY}
-        nodes:
-          - role: control-plane
-            kubeadmConfigPatches:
```

Comment:
Loss the config for ANP cert, it's the reason i change it to use config of `examples/kind/kind.config`,avoid missing something.